nunaliit2-mail/src/main/java/ca/carleton/gcrc/mail/MailDeliveryImpl.java

```diff
@@ -32,7 +32,15 @@ public MailDeliveryImpl() {
 
 	@Override
 	public boolean isConfigured() {
-		if( null == fromAddress ) {
+
+		String prot = mailProperties.getProperty("mail.transport.protocol",null);
+		String userName = mailProperties.getProperty("mail."+prot+".user",null);
```

Comment:
Unsure if mail. ... .user and .password are used as a part of mail configuration normally